src/shogun/labels/BinaryLabels.cpp

```diff
@@ -139,8 +141,28 @@ CLabels* CBinaryLabels::duplicate() const
 	return new CBinaryLabels(*this);
 }
 
-namespace shogun
-{
+
+namespace shogun {
+	SG_FORCED_INLINE Some<CBinaryLabels> from_multiclass(CMulticlassLabels* orig) {
+		auto result_vector = orig->get_labels();
+		std::set<int32_t> unique(result_vector.begin(), result_vector.end());
+		auto min = *std::min_element(unique.begin(), unique.end());
+
+		ASSERT(unique.size() == 2);
```

Comment:
User facing error msg pls

Reply:
I.e. in the lines of `REQUIRE("%s must contain less than two unique labels, contains %d. Cannot convert to binary labels.\n", orig->get_name(), unique.size());`